api/mapping/models.py

```diff
@@ -283,7 +294,7 @@ class ScanReportField(BaseModel):
         max_digits=10,
     )
 
-    ignore_column=models.CharField(
+    flag_column=models.CharField(
```

Comment:
This is the ScanReportField model, so I changed ignore_column to flag_column since "Flag"  is the name we chose for the Scan Report excel column, it used to be called "Ignore" which was a bit confusing. I thought it's best we keep the same naming across the Scan Report and the web app